src/detect-flowbits.c

```diff
@@ -387,6 +452,10 @@ int DetectFlowbitSetup (DetectEngineCtx *de_ctx, Signature *s, const char *rawst
             goto error;
     }
 
+    if (DetectFlowbitValidateCallback(s, cd) != 0) {
+        FatalError("invalid flowbit command combination in the same signature");
```

Comment:
this is not acceptable: it would mean a bad rule/ruleset can kill a running suri during a reload. We need to properly unwind things on error